grype/match/explicit_ignores.go

```diff
@@ -0,0 +1,42 @@
+package match
+
+import "github.com/anchore/grype/internal/log"
+
+type Ignore struct {
```

Comment:
since this is a helper type that utilizes mind reducing the scopes where this can be used? This can be done in a couple ways:
1. unexporting `Ignore` by renaming to `ignore`
2. further reduce the scope of both the `Ingore` type and `explicitIgnoreRules` variables by moving their definitions to within the `init` function